app/driver.go

```diff
@@ -319,16 +319,6 @@ type DaemonSetOps struct {
 	kubeClient *clientset.Clientset
 }
 
-func newDaemonSetOps(kubeClient *clientset.Clientset) (*DaemonSetOps, error) {
```

Comment:
After rechecking the code, it seems that struct `DaemonSetOps` is not used as well. I think you can remove the struct as well as all methods/functions for it.